ares/sfc/ppu-performance/io.cpp

```diff
@@ -1,5 +1,5 @@
 auto PPU::latchCounters() -> void {
-  cpu.synchronize(ppu);
+  cpu.synchronize(*this);
```

Comment:
in all these cases that pass a PPU pointer to the member function, would it be possible to somehow do this via the constructor instead? 

That way we can avoid modifying all these callers and keep the code looking a little more neat/tidy

Reply:
I rolled back the changes that removed references to globals.